hkmc2/shared/src/test/mlscript/rp/Future.mls

```diff
@@ -138,70 +132,68 @@ if input is Lines of Email then
 //│ ║  l.123: 	    "\n" ~ (Lines of L) as t then t
 //│ ║         	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 //│ ║  l.124: 	  L as h ~ Tail as t then [h, ...t]
-//│ ║         	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-//│ ║  l.125: 	  
-//│ ║         	^^^
-//│ ║  l.126: 	:todo
-//│ ╙──       	^^^^^
+//│ ╙──       	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:todo
+if input is Lines of Email then
+//│ ╔══[PARSE ERROR] Expected start of expression in this position; found end of input instead
+//│ ║  l.138: 	if input is Lines of Email then
+//│ ╙──       	                               ^
 //│ ╔══[ERROR] Name not found: input
-//│ ║  l.127: 	if input is Lines of Email then
+//│ ║  l.138: 	if input is Lines of Email then
 //│ ╙──       	   ^^^^^
 
 :todo
 pattern Opt(pattern P) = case
   P(value) then (Some(value))
   ""       then (None)
-  
+//│ ╔══[ERROR] Unrecognized pattern (case).
+//│ ║  l.148: 	  P(value) then (Some(value))
+//│ ║         	  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+//│ ║  l.149: 	  ""       then (None)
+//│ ╙──       	^^^^^^^^^^^^^^^^^^^^^^
+
 :todo
```

Comment:
I think this is parsed as
```fs
pattern Email(name, domain) = {
}
```

Reply:
Thanks. I marked that this is expected to be reported in 08036b903.

Reply:
Would be good to mention, in the comment, how this is parsed.

Reply:
Fixed in 2b722c98b.